coalib/parsing/StringProcessing/Core.py

```diff
@@ -328,6 +328,38 @@ def escape(string, escape_chars, escape_with="\\"):
     return string
 
 
+def convert_to_raw(string, exceptions=""):
+    """
+    Converts a string to its raw form, converting all backslash to double
+    backslash except when the backslash escapes a character given in
+    exceptions.
+
+    :param string:     The given string that needs to be converted
+    :param exceptions: A list of characters that if escaped with backslash
+                       should not be converted to double backslash.
+    :return:           Returns the corresponding raw string.
+    """
+    exceptions += "/"
```

Comment:
Yeah, it is never passed as an exception, but it should be. For example, consider the case where we `re.escape` a path. Then it becomes `\/tmp\/file` which should be interpreted as it is, not `\\/tmp\\/file`. But it is not a delimiter.